### PR TITLE
SPOI-10684 Kafka to Cassandra with transform application template.

### PR DESCRIPTION
**Pull request by deepak-narkhede**
Kafka to Cassandra with transform application template.

### CODE REVIEW

**Comment by deepak-narkhede** [comment]
@yogidevendra or @chaithu14 or @shubham-pathak22 either one of you can review.

**Comment by deepak-narkhede** [comment]
@patilvikram. Could you please review.

File: kafka-to-cassandra-transform/src/main/java/com/datatorrent/apps/Application.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
+/**
```
Comment (patilvikram) [inline kafka-to-cassandra-transform/src/main/java/com/datatorrent/apps/Application.java]: Do we need to add licensing block specific to Apache License? Isn't this one proprietary code?

File: kafka-to-cassandra-transform/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+### Description
+The Kafka to Cassandra transform application ingest messages in json string format from configured kafka topic, transforms the data and writes each message as a record in Cassandra Database store. This application uses PoJoEvent as an example schema, this can be customized to use custom schema based on specific needs.
```
Comment (patilvikram) [inline kafka-to-cassandra-transform/README.md]: "in json string format"  => " in JSON string format"
"PoJoEvent" => "POJOEvent"

File: kafka-to-cassandra-transform/README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+- The application is fault tolerant and can withstand node and cluster outages without data loss.
+- The application is also highly performant and can process as fast as the network allows.
+- It is extremely easy to add custom logic to get your business value without worrying about database connectivity and operational details of database writer.
+- The only configuration user needs to provide is source kafka broker lists and cassandra database details like node, table and keyspace.
```
Comment (patilvikram) [inline kafka-to-cassandra-transform/README.md]: Can you make this as bullet points
Such as 
Configuration required
1) Kafka Broker List ( property name )
2) Cassandra Database Details: 
    property string: purpose + example of the value .
    node : Identifies server node in Cassandra  ( eg. host url ) etc.

File: kafka-to-cassandra-transform/README.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+- This enterprise grade application template will dramatically reduce your time to market and cost of operations.
+
+Import the application from DataTorrent AppHub and launch it to ingest messages in json string format from configured kafka topic and writes each message as a record in cassandra store with transformation. Follow the tutorial videos or walkthrough document below to launch the template and add custom logic to process the data during ingestion.
+
```
Comment (patilvikram) [inline kafka-to-cassandra-transform/README.md]: json => JSON

Can user download App Template from somewhere if they do, please also mention URL to that location?

File: kafka-to-cassandra-transform/README.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+### Jumpstart
+Import and run application template as an operable proof of concept. Please watch the [walkthrough video](https://www.youtube.com/watch?v=) to import and launch the application.
```
Comment (patilvikram) [inline kafka-to-cassandra-transform/README.md]: No need to mention as "operable proof of concept"

Comment (patilvikram) [inline kafka-to-cassandra-transform/README.md]: "to import and launch the application." => "for more details "

File: kafka-to-cassandra-transform/README.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+### Launch App Properties
+
+Here is a preview of the properties to be set at application launch
+
+![Launch App Properties](https://www.datatorrent.com/wp-content/uploads/2016/12/)
+
+### Resources
+
+Please find the walkthrough docs for app template as follows:
+
```
Comment (patilvikram) [inline kafka-to-cassandra-transform/README.md]: No need to write "Please find"  
For **Walkthrough documentation** [ make this as a link ] for app template

Similarly make workd "GitHub URL" as link for the github

File: kafka-to-cassandra-transform/README.md (hunk 3 above)
Comment (deepak-narkhede) [inline kafka-to-cassandra-transform/README.md]: They are part of walkthrough docs. Please refer to other templates.

File: kafka-to-cassandra-transform/README.md (hunk 4 above)
Comment (deepak-narkhede) [inline kafka-to-cassandra-transform/README.md]: URL are updated only when the images are uploaded to site. They would be done after schema part is done.

File: kafka-to-cassandra-transform/README.md (hunk 2 above)
Comment (deepak-narkhede) [inline kafka-to-cassandra-transform/README.md]: Json done. For Pojo, Class name is PojoEvent. Hence changed to class name.

File: kafka-to-cassandra-transform/README.md (hunk 5 above)
Comment (deepak-narkhede) [inline kafka-to-cassandra-transform/README.md]: done.

File: kafka-to-cassandra-transform/README.md (hunk 6 above)
Comment (deepak-narkhede) [inline kafka-to-cassandra-transform/README.md]: Done.

File: kafka-to-cassandra-transform/src/main/java/com/datatorrent/apps/Application.java (hunk 1 above)
Comment (deepak-narkhede) [inline kafka-to-cassandra-transform/src/main/java/com/datatorrent/apps/Application.java]: Yes we need to add license. As the repo is under Apache license.

**Comment by deepak-narkhede** [comment]
@patilvikram . Incorporated the comments. Please have a look.

File: kafka-to-cassandra-transform/src/main/java/com/datatorrent/apps/Application.java (hunk 1 above)
Comment (patilvikram) [inline kafka-to-cassandra-transform/src/main/java/com/datatorrent/apps/Application.java]: Thanks for the explanation

File: kafka-to-cassandra-transform/README.md (hunk 3 above)
Comment (patilvikram) [inline kafka-to-cassandra-transform/README.md]: Ok, in that case, don't address comments I added here.

**Pull request merged**